examples_stardust/Cargo.toml

```diff
@@ -0,0 +1,54 @@
+[package]
+name = "utils"
+version = "0.6.0"
+authors = ["IOTA Stiftung"]
+edition = "2021"
+publish = false
+
+[dependencies]
+anyhow = "1.0.62"
+identity_core = { path = "../identity_core" }
+identity_did = { path = "../identity_did" }
+identity_stardust = { path = "../identity_stardust" }
+iota-client = { version = "2.0.0-beta.2", default-features = false, features = ["tls"] }
+primitive-types = "0.11.1"
+tokio = "1.20.1"
+
+[lib]
+path = "utils/utils.rs"
+
+[[example]]
+path = "crud/0_create_did.rs"
+name = "0_create_did"
```

Comment:
Seems like we would add or remove examples rather rarely, so fixing the numbers in those cases seems okay. I would stick with the global numbering.

Reply:
When we have three folders---`basic/crud`, `credentials`, `advanced`---it's going to be more annoying. Adding something like a new revocation example to `credentials` would mean incrementing all the `advanced` numbers. It's fine for now since there are just so few.